src/Main.js

```diff
@@ -13,7 +13,10 @@ export default {
     }
 
     if (opts.connectManually) {
-      Vue.prototype.$connect = () => {
+      Vue.prototype.$connect = (connectionAlternative) => {
```

Comment:
please use default value
`Vue.prototype.$connect = (connectionOptions = connection) => {`

Reply:
Will do

Reply:
tnx

Reply:
Would you prefer the default value replace the `if statement` and then uses `connectionAlternative` as the value that is passed to `observer = new Observer(connection, opts)` like such:

```
Vue.prototype.$connect = (connectionAlternative = connection) => {
        observer = new Observer(connectionAlternative, opts)
        Vue.prototype.$socket = observer.WebSocket
      }
```

Reply:
Done